sematic/ui/packages/common/src/pages/RunDetails/PipelineSection.tsx

```diff
@@ -43,29 +47,37 @@ const StyledPipelineTitle = styled(PipelineTitle)`
 `;
 
 const PipelineSection = () => {
+    const { rootRun } = useRootRunContext() as RemoveUndefined<ExtractContextType<typeof RootRunContext>>;
+
+    const { error, avgRuntime, successRate, totalCount }
+        = useBasicMetrics({ runId: rootRun.id, rootFunctionPath: rootRun.function_path });
+
     return (
         <>
             <TopSection>
                 <Headline>Pipeline</Headline>
-                <StyledPipelineTitle>MNIST PyTorch Example</StyledPipelineTitle>
+                <StyledPipelineTitle>{rootRun.name}</StyledPipelineTitle>
                 <ImportPath>
-          sematic.examples.mnist.pipeline.with.very.long.path
+                    {rootRun.function_path}
                 </ImportPath>
                 <StyledVertButton />
             </TopSection>
             <BottomSection>
-                <Box>
-                    <Typograph variant="bigBold">257</Typograph>
-                    <Typograph variant="body1">runs</Typograph>
-                </Box>
-                <Box>
-                    <Typograph variant="bigBold">74 min.</Typograph>
-                    <Typograph variant="body1">avg. runtime</Typograph>
-                </Box>
-                <Box>
-                    <Typograph variant="bigBold">62%</Typograph>
-                    <Typograph variant="body1">success</Typograph>
-                </Box>
+                {error && <Alert severity="error">{error.message}</Alert>}
+                {!error && <>
+                    <Box>
+                        <Typograph variant="bigBold">{totalCount}</Typograph>
+                        <Typograph variant="body1">runs</Typograph>
+                    </Box>
+                    <Box>
+                        <Typograph variant="bigBold">{`${avgRuntime}`}</Typograph>
+                        <Typograph variant="body1">avg. runtime</Typograph>
+                    </Box>
+                    <Box>
+                        <Typograph variant="bigBold">{successRate}</Typograph>
+                        <Typograph variant="body1">success</Typograph>
+                    </Box>
+                </>}
```

Comment:
What happens while the query is loading? Maybe we should use `Skeleton`.

Reply:
Cool. Skeleton is adopted.